spec/support/capybara.rb

```diff
@@ -8,7 +8,12 @@
   config.include Capybara::DSL
 end
 
-base_args = %w[headless no-sandbox disable-gpu]
+if ENV['NO_HEADLESS'] == '1'
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Style/ConditionalAssignment: Use the return of the conditional for variable assignment and comparison.